### ISSUE TITLE
Installation fails with latest pip version 20.2.3

### ISSUE THREAD

**Issue by vshymanskyy** [wasienv/wasienv#32]
Encountered on Github Actions runner.
```sh
> Installing wasienv
WARNING: pip is being invoked by an old script wrapper. This will fail in a future version of pip.
Please see https://github.com/pypa/pip/issues/5599 for advice on fixing the underlying issue.
To avoid this problem you can invoke Python with '-m pip' instead of running pip directly.
WARNING: Skipping wasienv as it is not installed.
WARNING: pip is being invoked by an old script wrapper. This will fail in a future version of pip.
Please see https://github.com/pypa/pip/issues/5599 for advice on fixing the underlying issue.
To avoid this problem you can invoke Python with '-m pip' instead of running pip directly.
/home/runner/.local/lib/python3.6/site-packages/pip/_internal/commands/install.py:235: UserWarning: Disabling all use of wheels due to the use of --build-option / --global-option / --install-option.
  cmdoptions.check_install_build_global(options)
ERROR: Location-changing options found in --install-option: ['--install-scripts'] from command line. This is unsupported, use pip-level options like --user, --prefix, --root, and --target instead.
```

Everything works after downgrading to pip v20.1.1

**Comment by tomzhang** [comment]
this error message:
ERROR: Location-changing options found in --install-option: ['--install-scripts'] from command line. This is unsupported, use pip-level options like --user, --prefix, --root, and --target instead.
already shows up that there is  unsupported arguments in your current pip environment.

**Comment by artlogic** [comment]
This appears to have been fixed by 1cb32a0d62ac9ae97ac34981ca71717b4dfd7f27
